extension/popup.html

```diff
@@ -0,0 +1,37 @@
+ <html>
+    <head>
+        <link rel="stylesheet" href="./options/fonts/material-icons/material-icons.css">
+
+        <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
+
+        <link rel="stylesheet" type="text/css" href="popup.css"/>
+
+
+    </head>
+    <body>
+        <div>
+
+              <form method="get" action="overview/overview.html?query=" target="_blank">
+                <i class="material-icons">search</i>
+                <input id="search" name="query" placeholder="Search" target="_blank" />
+              </form>
+
+              <a href="options/options.html#/settings" target="_blank">
+                <i class="material-icons">settings</i>
+                Settings</a>
+
+              <a href="https://www.reddit.com/r/WorldBrain/" target="_blank">
+              <i class="material-icons">feedback</i>
+              Feedback</a>
+<!--
+              <br/> -->
+
+              <a href="options/options.html#/import" target="_blank">
+              <i class="material-icons">file_download</i>
+              Import History &amp; Bookmarks</a>
```

Comment:
I think if we have the whole thing looking more like the lastpass dropdown(as seen [here](https://github.com/WorldBrain/WebMemex/issues/22)), the width should not be an issue anymore. Now the whole thing is more dense than it could/should be. 

Not sure about all caps either.